common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/dialog/RadioListAlertDialogBuilder.kt

```diff
@@ -171,19 +174,58 @@ class RadioListAlertDialogBuilder(val context: Context) : DaxAlertDialog {
             setOnCheckedChangeListener { _, checkedId ->
                 listener.onRadioItemSelected(checkedId)
                 binding.radioListDialogPositiveButton.isEnabled = true
+                binding.radioListDialogDestructivePositiveButton.isEnabled = true
             }
         }
 
-        binding.radioListDialogPositiveButton.text = positiveButtonText
-        binding.radioListDialogPositiveButton.setOnClickListener {
-            listener.onPositiveButtonClicked(binding.radioListDialogRadioGroup.checkedRadioButtonId)
-            dialog.dismiss()
-        }
+        setButtons(binding, dialog)
+    }
+
+    private fun setButtons(
+        binding: DialogSingleChoiceAlertBinding,
+        dialog: AlertDialog,
+    ) {
+        binding.radioListDialogPositiveButton.isVisible = !isDestructiveVersion
         binding.radioListDialogPositiveButton.isEnabled = selectedOption != null
+        binding.radioListDialogDestructivePositiveButton.isVisible = isDestructiveVersion
+        binding.radioListDialogDestructivePositiveButton.isEnabled = selectedOption != null
+        binding.radioListDialogNegativeButton.isVisible = !isDestructiveVersion
+        binding.radioListDestructiveDialogNegativeButton.isVisible = isDestructiveVersion
+
+        if (isDestructiveVersion) {
+            setButtonListener(
+                binding.radioListDialogDestructivePositiveButton,
+                positiveButtonText,
+                dialog,
+            ) { listener.onPositiveButtonClicked(binding.radioListDialogRadioGroup.checkedRadioButtonId) }
+            setButtonListener(binding.radioListDestructiveDialogNegativeButton, negativeButtonText, dialog) { listener.onNegativeButtonClicked() }
+            // no need to get fancy, just change the button textColor
+            binding.radioListDestructiveDialogNegativeButton.setTextColor(
+                ContextCompat.getColorStateList(
+                    context,
+                    R.color.secondary_text_color_selector,
+                ),
```

Comment:
This code needs to be refactored as this should be "Ghost Alt"(what a name) which is in the new [Figma Components file](https://www.figma.com/file/BOHDESHODUXK7wSRNBOHdu/Android-Components?node-id=6550-28349&t=Y957CnfKdGNZ2W6l-4) instead setting the color directly in the code.
I'll create the task for working on it in a separate branch 👍